packages/strapi-helper-plugin/lib/src/components/PopUpWarning/StyledBody.js

```diff
@@ -2,98 +2,33 @@ import styled from 'styled-components';
 import { ModalBody } from 'reactstrap';
 
 const StyledBody = styled(ModalBody)`
-  padding: 16px 30px 15px 30px !important;
-
+  padding: 0;
   .modalBodyContainerHelper {
```

Comment:
why about creating new components here (ModalBodyContainerHelper, PopUpWarningButtonContainer, etc)

Reply:
I also like the idea!

Reply:
I agree too and I change it with pleasure.